account_avatax/models/account_move.py

```diff
@@ -138,10 +138,8 @@ def _avatax_prepare_lines(self, doc_type=None):
     def _avatax_compute_tax(self, commit=False):
         """ Contact REST API and recompute taxes for a Sale Order """
         self and self.ensure_one()
+        Tax = self.env["account.tax"]
         avatax_config = self.company_id.get_avatax_config_company()
-        if not avatax_config:
-            # Skip Avatax computation if no configuration is found
-            return
```

Comment:
You'll get errors when running Odoo tests if you remove this.

Reply:
ah will re-add, I have that in my migration.yml